src/BankingApp.Data/Entities/Transaction.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BankingApp.Data.Entities
+{
+    public class Transaction : BaseEntity
+    {
+        public int TransactionId { get; set; }
+        public DateTime Date { get; set; }
+        public decimal Amount { get; set; }
+        public TransactionType Type { get; set; }
+        public string Description { get; set; }
+
```

Comment:
There should also be a reference field too. Please think through the possible fields again and lets discuss